manticore/platforms/evm.py

```diff
@@ -1037,7 +1037,7 @@ def setstate(state, value):
             def setstate(state, value):
                 current_vm = state.platform.current_vm
                 _pc, _old_gas, _instruction, _arguments, _fee, _allocated = current_vm._checkpoint_data
-                current_vm._checkpoint_data = (_pc, _old_gas, _instruction, _arguments, _value, _allocated)
+                current_vm._checkpoint_data = (_pc, _old_gas, _instruction, _arguments, _fee, _allocated)
```

Comment:
`_value` to `value` not to `fee`.
This need to change the fee part of `current_vm._checkpoint_data` to `value`